### PR TITLE
Set version

### PR DESCRIPTION
**Pull request by iwahbe**
Fixes https://github.com/pulumi/pulumi-cloudflare/issues/403

### CODE REVIEW

**Comment by t0yv0** [comment]
I am really curious why this generates schema changes. Is it an example of non-hermetic updates somewhere?

**Comment by t0yv0** [comment]
Confirming that this fixes

```
make provider  && ./bin/pulumi-resource-cloudflare                                                                                                                                                                                                                                          51564
```

Which used to fail. This would've been detected if acc-tests were not disabled on this repo.

File: examples/examples_py_test.go [hunk 1]
```diff
@@ -12,8 +12,6 @@ import (
 )
 
 func TestAccRecordPy(t *testing.T) {
-	t.Skip("Temporarily skip while releasing new major version")
```
Comment (t0yv0) [inline examples/examples_py_test.go:15]: Nice catch.

**Review by t0yv0 — APPROVED** [review]
Definitely fixes the P1 issue! 

Still got a few open questions but can follow up later. I'm for merging if we get a passing build.

**Comment by t0yv0** [comment]
Ok we found out why schema changes, autoalising is dropping MaxItemOne=1 withholding due to a major version upgrade, as expected.. but it was not kicking in due to Version not being set, until this PR. This is better then. Let's look at why the tests are failing.

**Comment by t0yv0** [comment]
Ah we just need `make build_sdks`

**Comment by iwahbe** [comment]
> Ah we just need `make build_sdks`

We don't actually want to take these changes since it isn't actually a major version bump.

**Comment by t0yv0** [comment]
So to clarify, should not those changes have been made at 5.x.x release but weren't made because of the Version propagation issue?

**Comment by iwahbe** [comment]
> So to clarify, should not those changes have been made at 5.x.x release but weren't made because of the Version propagation issue?

Those changes would have ideally been made at the 5.0.0 release, but weren't made because the 5.0.0 release predated `x.AutoAliasing` (which is what looks at the version).

File: provider/cmd/pulumi-resource-cloudflare/bridge-metadata.json [hunk 2]
```diff
@@ -1956,10 +2048,12 @@
                 }
             },
             "cloudflare_zone_dnssec": {
-                "current": "cloudflare:index/zoneDnssec:ZoneDnssec"
+                "current": "cloudflare:index/zoneDnssec:ZoneDnssec",
+                "majorVersion": 5
             },
             "cloudflare_zone_lockdown": {
                 "current": "cloudflare:index/zoneLockdown:ZoneLockdown",
+                "majorVersion": 5,
```
Comment (t0yv0) [inline provider/cmd/pulumi-resource-cloudflare/bridge-metadata.json:2056]: Hmm so should these majorVersion: 5 changes be pushed back onto the branch to inform auto-aliasing major version heuristic better?

**Pull request merged**